mssqlcli/mssql_cli.py

```diff
@@ -302,7 +306,7 @@ def handle_editor_command(self, text):
             editor_command = special.editor_command(text)
         return text
 
-    def execute_command(self, text, query):
+    def execute_command(self, text, query, test_format=False):
```

Comment:
@pensivebrian pointing this change out for your awareness—this optional param helps a lot with testing but I'm very open to alternative methods you know would work that don't require extensive changes to `execute_command`.